style-sheets/primevue-components-overrides.scss

```diff
@@ -1140,4 +1140,133 @@ label {
         display: block;
         margin-top: 1.5rem;
     }
+}
+
+/* ------------------------------------ tabView ----------------------------- */
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-panels {
+    padding: 0;
+    background-color: transparent;
+    z-index: 1;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav {
+    border: none;
+    position: absolute;
```

Comment:
also, we shouldn't set the position here. I get what you're trying to do, so set this to be absolute, and in FAM, you have the tabview "panelContainer" margin top so the content won't be hide. But this is not the correct way, we shouldn't do this.

Reply:
I'll do in a different way